### ISSUE TITLE
Test not finishing

### ISSUE THREAD

**Issue by maxulysse** [askimed/nf-test#110]
Hi,
I'm running this test https://github.com/nf-core/fetchngs/blob/57bf16705bf23fe2c7695774e2376f2c709a4d35/workflows/sra/tests/sra_skip_fastq_download.nf.test

It should be fairly similar to running this command `nextflow run . -profile test,docker --outdir results -resume --skip_fastq_download`, which takes less than a couple of minutes, but I can never seem to make it finish (cf https://github.com/nf-core/fetchngs/actions/runs/6000651325/job/16273188355?pr=191).

I did try some investigation on my machine but could not find out why the test would hang.
I had a look at the trace, and all tasks are completed.

Any idea where I can investigate more?

**Comment by lukfor** [comment]
Thanks, sounds related to #57 🤔

**Comment by maxulysse** [comment]
indeed, it looks related, sorry I did not notice that issue.

**Comment by lukfor** [comment]
No worries. Good to have an other reproducible usecase! please let the issue open, so we know that it is critical for nf-core 👍

**Comment by maxulysse** [comment]
Yeah, I'm really trying to fully replace the current tests we have with pytest by nf-test.
I do love this testing framework, just the snapshot possibilities are amazing.
Also, I really like how easy it is to test Nextflow at any level.

**Comment by lukfor** [comment]
PR #112 resolves issue #57. This PR should also address this issue. I'm leaving the issue open until we create a new release and confirm its resolution.

**Comment by lukfor** [comment]
We have just released RC1 of version 0.8.0 :tada:
This is a pre-release and can be installed using the following command:
```
curl -fsSL https://code.askimed.com/install/nf-test | bash -s 0.8.0-rc1
```
If everything works as expected, please feel free to close this issue. Thanks for your support! :pray:

**Comment by maxulysse** [comment]
works as expected
